src/main/java/org/team1540/robot2024/subsystems/shooter/Shooter.java

```diff
@@ -261,12 +260,12 @@ public Command spinUpAndSetPivotPosition(Supplier<Double> leftSetpoint, Supplier
         );
     }
 
-    @AutoLogOutput
+    @AutoLogOutput(key = "Shooter/Flywheels/leftSetpointRPM")
```

Comment:
ok to be clear, we now have AdvantageScope/RealOutputs/Shooter/Flywheels, AdvantageScope/Shooter/Flywheels, SmartDashboard/Shooter/Flywheels, and SmartDashboard/TunableNumbers/Shooter/Flywheels. May be useful to think about consolidating some of those

Reply:
Each of these tables represents a different use case of publishing to NT and/or logs. AdvantageKit/RealOutputs/Shooter/Flywheels and AdvantageKit/Shooter/Flywheels are only for logging of inputs and outputs, and no user dashboard inputs should come from these. SmartDashboard/Shooter/Flywheels should only be used for values that would like to be changed conveniently at any point, whereas SmartDashboard/TunableNumbers/Shooter/Flywheels are for values that should be constant during normal code execution, but we want to have access to if we need to tune them.